README.md

```diff
@@ -120,7 +120,7 @@ $ k8t new cluster MyCluster
 Create a new environment
 
 ```bash
-$ k8t new environment staging
+$ k8t new environment staging -c MyCluster
```

Comment:
actually both are possible, the idea being that you could globally define something for the staging environment as well as for the staging environment in a particular cluster

Reply:
TBH I feel we need kinda flow here